tests/NumericRatingTest.php

```diff
@@ -42,7 +46,38 @@ public function testNumericCanGetAverage() {
     );
     $average = $this->numericRating->getAverage($ratingsList);
     $this->assertEquals(3,$average);
-    echo ($average);
```

Comment:
Since this affected the output of the tests, I've removed this, but if you'd prefer I can revert this line.